.github/ISSUE_TEMPLATE/credentials-access.md

```diff
@@ -0,0 +1,25 @@
+--
+name: Credential Store Access Request
+about: Request access to the OpenTelemetry credential store
+title: 'REQUEST: Credential Store Access for <USERNAME>'
+labels: area/repo-maintenance
+assignees: ''
+---
+
+<!-- Please remember to change the title of this issue by replacing
+ <USERNAME> with the actual github username -->
+
+### Repository Vault Requested
```

Comment:
Should we also add a section where the purpose for gaining access is described?

Reply:
I suspect most vaults will be SIG-based, and it would just be the SIG maintainers requesting access.